### PR TITLE
feat(just): Created 70-nix.just with nix installation, uninstallation, and configuration

### PR DESCRIPTION
**Pull request by dnkmmr69420**
there were quite a bit of nix stuff added possibly with more nix stuff added

### CODE REVIEW

**Comment by KyleGospo** [comment]
Not every image has that imageinfo file, maybe check for the presence of mutter instead.

**Comment by dnkmmr69420** [comment]
> Not every image has that imageinfo file, maybe check for the presence of mutter instead.

can you submit a review for that so I can just press a button for it to be applied. I have no idea how to do it.

**Comment by dnkmmr69420** [comment]
> Not every image has that imageinfo file, maybe check for the presence of mutter instead.

I made it hidden for now. Can I get this merged pls?

**Review by EyeCantCU — APPROVED** [review]

File: build/ublue-os-just/70-nix.just [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if [[ -d "/nix" ]]; then
+      mkdir -p $HOME/.config/nix
+      echo "experimental-features = nix-command flakes" >> $HOME/.config/nix/nix.conf
+      curl -fsSL https://getfleek.dev/installer | env FORCE=1 bash
+    else
+      echo "Install nix with ujust nix-install before running ujust fleek-install."
+    fi
+
+# Integrate Nix/Fleek applications with system theme
+fleek_themes_install:
```
Comment (nadavspi) [inline build/ublue-os-just/70-nix.just:39]: Any reason why this one is snake case when the others use dashes?

Comment (EyeCantCU) [reply build/ublue-os-just/70-nix.just:39]: No reason. Will fix that

**Review by KyleGospo — APPROVED** [review]

**Pull request merged**